e2e_tests/tests/kyt-build.test.js

```diff
@@ -20,7 +20,7 @@ describe('kyt build', () => {
     expect(shell.test('-f', 'build/public/nothing.txt')).toBe(true);
 
     // Should produce the manifest, main and vendor scripts
-    expect(shell.ls('build/public/manifest-*.js').code).toBe(0);
+    expect(shell.ls('build/public/runtime~main-*.js').code).toBe(0);
```

Comment:
We used to inline this internally (`KYT.ASSETS_MANIFEST`) -- would that still be necessary/possible?

Reply:
Yes - can still be plucked out by clientAssets[‘runtime~main.js’], I’ll make a Vi patch to illustrate